fontforge/python.c

```diff
@@ -1751,7 +1753,20 @@ return( reto );
 /* Points */
 /* ************************************************************************** */
 
-static PyFF_Point *PyFFPoint_CNew(double x, double y, int on_curve, int sel, char *name) {
+static const char *py_point_types[] = { "splineCorner", "splineCurve", "splineHVCurve",
+				     "splineTangent", NULL };
+// point types: see 'enum pointtype' in splinefont.h
+static const int typemap_from_python[] = { 1, 0, 3, 2 };
```

Comment:
I would be very uncomfortable touching the original enum. The reason for the reordering is to make corner be 0 to support `smooth` and so that `calloc` gives the appropriate default.